assets/js/admin/wc-product-import.js

```diff
@@ -0,0 +1,78 @@
+/*global ajaxurl, wc_product_import_params */
+;(function ( $, window ) {
+
+	/**
+	 * productImportForm handles the import process.
+	 */
+	var productImportForm = function( $form ) {
+		this.$form = $form;
+		this.xhr   = false;
+		this.mapping = wc_product_import_params.mapping;
+		this.position = 0;
+		this.file = wc_product_import_params.file;
+		this.security = wc_product_import_params.import_nonce;
+
+		// Number of import successes/failures.
+		this.imported = 0;
+		this.failed = 0;
```

Comment:
Why can't we keep track of this in the JS file and have the ajax endpoint return those values? We just need to SUm, then pass to done step.